src/README.md

```diff
@@ -0,0 +1,27 @@
+## Welcome to the Babel Codebase
+
+Babel is broken down into three parts:
+
+- Parsing
+- Transformation
+- Generation
+
+**Parsing** is the process of turning a piece of code into an AST (Abstract
+Syntax Tree) which is a tree-like object of nodes that _describes_ the code.
+This makes it easier to transform the code over direct string manpulation.
+
+**Transformation** is the process of taking an AST and manipulating it into
+a new one. For example, Babel might take an ES2015 ArrowFunction and transform
```

Comment:
I went we these because they have a bit more context, and hopefully are less likely to be ignored.

Also, [GitHub will display them as you browse the repo](https://github.com/babel/babel/tree/4d4ac406691c066060f4fccd35092070eeb5c124/src).